sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_patch.py

```diff
@@ -6,9 +6,60 @@
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import List
+from typing import Any, List, Optional, Union
 
-__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+
+from ._client import(
+    DocumentIntelligenceClient as DIClientGenerated,
+    DocumentIntelligenceAdministrationClient as DIAClientGenerated,
+)
+
+
+class DocumentIntelligenceClient(DIClientGenerated):  # pylint: disable=client-accepts-api-version-keyword
+    # It's a bug in Pylint checker: https://github.com/Azure/azure-sdk-tools/issues/7437
+    __doc__ = DIClientGenerated.__doc__  # pylint: disable=client-incorrect-naming-convention
+
+    def __init__(
+        self,
+        endpoint: str,
+        credential: Union[AzureKeyCredential, TokenCredential],
+        *,
+        polling_interval: Optional[str] = 5, # The default polling interval should be 5 seconds
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(
+            endpoint=endpoint,
+            credential=credential,
+            polling_interval=polling_interval,
+            **kwargs,
+        )
+
+
+class DocumentIntelligenceAdministrationClient(DIAClientGenerated):  # pylint: disable=client-accepts-api-version-keyword
+    # It's a bug in Pylint checker: https://github.com/Azure/azure-sdk-tools/issues/7437
+    __doc__ = DIAClientGenerated.__doc__  # pylint: disable=client-incorrect-naming-convention
```

Comment:
This will not address the fact that intelligence and development-time friends won't see the doc comments. Only runtime introspection will work here. Which is - apart from auto doc usage (if this is documented using auto doc) - not that important.